README.md

```diff
@@ -48,6 +48,7 @@ Non-production plugins built by Mattermost staff and community.
 - [mattermost-plugin-profanity-filter](https://github.com/mattermost/mattermost-plugin-profanity-filter)
 - [mattermost-plugin-spoiler](https://github.com/moussetc/mattermost-plugin-spoiler)
 - [mattermost-plugin-statuspage](https://github.com/cpanato/mattermost-plugin-statuspage)
+- [mattermost-plugin-spybot](https://github.com/scottleedavis/mattermost-plugin-spybot)
```

Comment:
Would you please move this one up? Otherwise this is the wrong order.

Reply:
Done.  👍